pimd/pim6_cmd.c

```diff
@@ -324,6 +324,78 @@ DEFUN (show_ipv6_mroute_count_vrf_all,
 	return CMD_SUCCESS;
 }
 
+DEFUN (show_ipv6_mroute_summary,
+       show_ipv6_mroute_summary_cmd,
+       "show ipv6 mroute [vrf NAME] summary [json]",
+       SHOW_STR
+       IPV6_STR
+       MROUTE_STR
+       VRF_CMD_HELP_STR
+       "Summary of all mroutes\n"
+       JSON_STR)
+{
+	int idx = 2;
+	bool uj = use_json(argc, argv);
+	struct vrf *vrf = pim_cmd_lookup_vrf(vty, argv, argc, &idx);
+	json_object *json = NULL;
+
+	if (uj)
+		json = json_object_new_object();
+
+	if (!vrf)
+		return CMD_WARNING;
+
+	show_mroute_summary(vrf->info, vty, json);
```

Comment:
we need to check this code and correct few things for ipv6. We can't use the whole code as is since mfcctl will be different.